### PR TITLE
boards: Update STM32 disco boards default configuration

### PR DESCRIPTION
**Pull request by erwango**
First batch of patches to address #9247.

Impacted boards:
-stm32l496g_disco
-disco_l475_iot1
-stm32l476g_disco
-stm32f769i_disco
-stm32f723e_disco
-stm32f469i_disco
-stm32f429i_disco

### CODE REVIEW

**Review by ydamigos — CHANGES_REQUESTED** [review]
Looks good. Some minor comments.

File: boards/arm/stm32f469i_disco/doc/stm32f469i_disco.rst [hunk 1]
```diff
@@ -115,10 +115,12 @@ For mode details please refer to `32F469IDISCOVERY board User Manual`_.
...
 Default Zephyr Peripheral Mapping:
 ----------------------------------
-- UART_3_TX : PB10
-- UART_3_RX : PB11
-- UART_6_TX : PG14
-- UART_6_RX : PG9
+- UART_3 TX/RX : PB10/PB11 (ST-Link Virtual Port Com)
+- UART_6 TX/RX : PG14/PG9 (Arduino Serial)
+- I2C1 SCL/SDA : PB8/PB9 (Arduino I2C)
+- SPI1 NSS/SCK/MISO/MOSI : PA4/PA5/PA6/PA7 (Arduino SPI)
```
[the remark this answers is not in the document]
Comment (erwango) [reply boards/arm/stm32f469i_disco/doc/stm32f469i_disco.rst]: Fixed

File: boards/arm/stm32f769i_disco/doc/stm32f769i_disco.rst [hunk 2]
```diff
@@ -126,10 +130,11 @@ For mode details please refer to `32F769I-DISCO board User Manual`_.
...
 ----------------------------------
-- UART_1_TX : PA9
-- UART_1_RX : PA10
-- UART_6_TX : PC6
-- UART_6_RX : PC7
+
+- UART_1 TX/RX : PA9/PA10 (ST-Link Virtual Port Com)
+- UART_6 TX/RX : PC6/PC7 (Arduino Serial)
+- I2C1 SCL/SDA : PB8/PB9 (Arduino I2C)
+- SPI1 SCK/MISO/MOSI : PA12/PB14/PB15 (Arduino SPI)
```
[the remark this answers is not in the document]
Comment (erwango) [reply boards/arm/stm32f769i_disco/doc/stm32f769i_disco.rst]: Fixed

**Comment by erwango** [comment]
@ydamigos , comments fixed. Thanks for the review

**Review by dbkinder — APPROVED** [review]
doc changes look OK.

**Review by ydamigos — APPROVED** [review]

**Review by nashif — APPROVED** [review]

**Pull request merged**